src/libs/feature-previews-config.ts

```diff
@@ -4,7 +4,7 @@ export const HAIL_BATCH_AZURE_FEATURE_ID = 'hail-batch-azure';
 export const ENABLE_AZURE_PFB_IMPORT = 'enableAzurePfbImport';
 export const ENABLE_AZURE_TDR_IMPORT = 'enableAzureTdrImport';
 export const FIRECLOUD_UI_MIGRATION = 'firecloudUiMigration';
-export const AXIN_DATASET_CARD = 'axinDatasetCard';
+export const COHORT_BUILDER_CARD = 'cohortBuilderCard';
```

Comment:
Do we need to keep this feature setting in place? For demos, we can continue to use a bookmarked URL. Right now we don't have any test users of the cohort builder outside of the broad so I don't think the card and this setting is necessary any more.

Reply:
I asked Victoria if she'd still like it for demos, and she said yes, which is why I left it.